src/components/views/rooms/ReadReceiptGroup.tsx

```diff
@@ -203,7 +204,7 @@ function ReadReceiptPerson({ userId, roomMember, ts, isTwelveHour, onAfterClick
             onClick={() => {
                 dis.dispatch({
                     action: Action.ViewUser,
-                    member: roomMember,
+                    member: roomMember ?? { userId } as User,
```

Comment:
please copy the XXX: comment from the other call site as this is very much a smell which should be fixed in the future